src/main/python/hydra/zmqtest/runtest.py

```diff
@@ -26,42 +28,111 @@ def __init__(self, argv):
             config_fn = argv[1]
             del argv[1]
         RunTestBase.__init__(self, 'zmqScale', config, config_fn)
-
-        zstpub = '/zst-pub'
-        zstsub = '/zst-sub'
-        self.add_appid(zstpub)
-        self.add_appid(zstsub)
+        self.zstpub = '/zst-pub'
+        self.zstsub = '/zst-sub'
+        self.add_appid(self.zstpub)
+        self.add_appid(self.zstsub)
+        #self.total_sub_apps = 200
+        self.total_sub_apps = zmq_config.total_apps
         self.start_init()
+        self.run_test()
+
+    def run_test(self):
+        # Launch zmq pub
```

Comment:
can you put some arguments here which can then be picked up and passed to pub/sub.
message count, delay etc.